commit ce1a67def64d7eb4c0c364573ff1526713fda607
Author: popcornmix <popcornmix@gmail.com>
Date:   Tue Jun 13 15:06:22 2017 +0100

    Revert "drm/vc4: Add FB modifier support to firmwarekms."
    
    This reverts commit 9183dfdbf2cb3ec8e981204847b345f7ab95662d.

diff --git a/drivers/gpu/drm/vc4/vc4_firmware_kms.c b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
index 40a0cdc179ce..943cbb38f1f2 100644
--- a/drivers/gpu/drm/vc4/vc4_firmware_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
@@ -18,7 +18,6 @@
 #include "drm_atomic_helper.h"
 #include "drm_plane_helper.h"
 #include "drm_crtc_helper.h"
-#include "drm_fourcc.h"
 #include "linux/clk.h"
 #include "linux/debugfs.h"
 #include "drm_fb_cma_helper.h"
@@ -137,10 +136,6 @@ static void vc4_primary_plane_atomic_update(struct drm_plane *plane,
 	fbinfo->yoffset = state->crtc_y;
 	fbinfo->base = bo->paddr + fb->offsets[0];
 	fbinfo->pitch = fb->pitches[0];
-
-	if (fb->modifier[0] == DRM_FORMAT_MOD_BROADCOM_VC4_T_TILED)
-		fbinfo->bpp |= BIT(31);
-
 	/* A bug in the firmware makes it so that if the fb->base is
 	 * set to nonzero, the configured pitch gets overwritten with
 	 * the previous pitch.  So, to get the configured pitch
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1566_ce1a67def64d7eb4c0c364573ff1526713fda607.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1566_ce1a67def64d7eb4c0c364573ff1526713fda607.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
